apps/condo/domains/user/utils/directAccess/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Set of schemas available for direct access by specific users, similar to support / admin.
+ *
+ * Note that adding a schema to this config will automatically add the required fields to the UserRightsSet schema,
+ * but accesses will not work automatically!
+ * To do this, you need to explicitly add one of the utilities from "@condo/domains/user/utils/access" to the model accesses.
+ *
+ * Also note that changing this config always generates a new migration!
+ * @typedef {{schemaName: string, readonly?: boolean}} DirectAccessList
+ * @typedef {{lists: Array<DirectAccessList | string>, services: Array<string>}} DirectAccessConfig
+ * @type {DirectAccessConfig}
+ */
+const DIRECT_ACCESS_AVAILABLE_SCHEMAS = {
```

Comment:
Is it a task related list of available schemas? What if someone want to change this list? Did you thought about making that configurable not from a code source

Reply:
I did not get this question. You need to do 2 steps to add new schemas / delete schemas / modify config:

1. Change the config, so it will change set of fields in `UserRightsSet` (false for new by default for adding).
2. Generate a migration, since list of fields will be changed
3. Use one of the provided utils in access (that's a design decision, since we want all accesses to specified explicitly)

Reply:
And all of that info is described in JSDoc